rskj-core/src/main/java/co/rsk/peg/ErpRedeemScriptBuilderUtils.java

```diff
@@ -0,0 +1,51 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptChunk;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static co.rsk.peg.ErpFederationCreationException.Reason.INVALID_INTERNAL_REDEEM_SCRIPTS;
+import static co.rsk.peg.ErpFederationCreationException.Reason.INVALID_CSV_VALUE;
+
+public class ErpRedeemScriptBuilderUtils {
```

Comment:
For another ticket, but we are gonna need tests for this class

Reply:
I'll create it in the backlog